Example/AztecUITests/HighPriorityIssuesTests.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  HighPriorityIssuesTests.swift
+//  AztecUITests
+//
+//  Created by brbrr on 11/10/17.
+//  Copyright © 2017 Automattic Inc. All rights reserved.
+//
+
```

Comment:
You can remove these default header comments from the top of the file.

Reply:
✅